src/modules/logview/LogViewWindow.cpp

```diff
@@ -386,10 +391,11 @@ void LogViewWindow::filterNext()
 	new LogListViewLog(m_pLastGroupItem, pFile->type(), pFile);
 
 filter_next:
-	pFile = m_logList.next();
+	++m_currentLog;
+	pFile = *m_currentLog;
```

Comment:
This actually segfaults when at the last element.
In std::vector, end() returns the not allocated "past the last" element, that can't be dereferenced

Reply:
fixed in https://github.com/kvirc/KVIrc/pull/2485/commits/31cc120f3ba988fbd7f31a4c5db684c641c3de2e